### PR TITLE
Updated interactives documentation

### PR DESCRIPTION
**Pull request by SiAdcock**
## What does this change?

Documentation for interactives is currently sparse, out of date, broken and wrong.

After a lot of discussion around how interactives work, how many different forms of interactives we support and what long term support for existing and new interactives could look like going forward, it was decided to bring the interactives documentation up to date.
## What is the value of this and can you measure success?

Up-to-date documentation will allow us to move forward with interactives and decide how we are going to support legacy formats.
## Request for comment

@wpf500 @rich-harris @sndrs @guardian/dotcom-platform @stephanfowler

### CODE REVIEW

**Comment by Rich-Harris** [comment]
Excellent! Should we add a couple of pars around service/web workers? (ref https://github.com/guardian/frontend/pull/14762)

**Comment by SiAdcock** [comment]
Good idea @Rich-Harris!

File: docs/03-dev-howtos/05-interactives.md [hunk 1]
```diff
@@ -34,57 +58,42 @@ define(['your/dependencies'], function (dependency) {
     });
 ```
 
-## Embedding the interactive in a page
+## Interactive content atoms
+
+Interactive content atoms are the recommended way to create interactive content going forward. They are treated as first
```
Comment (sndrs) [inline docs/03-dev-howtos/05-interactives.md]: should this be at the top maybe? with a mention of deprecating the other two methods?

**Comment by sndrs** [comment]
❤️

**Comment by SiAdcock** [comment]
I've made a change to the explanation of custom boot scripts and articles , @wpf500. Do you reckon you could have a read of the sections "Custom Boot Scripts", "Interactive Article" and "Immersive Article" and see if they are more accurate?

**Review by gtrufitt — APPROVED** [review]
Nice work! 💟

**Pull request merged**